CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # bedrock-kms ChangeLog
 
+## 7.3.0 - TBD
```

Comment:
This doesn't add any API surface, it just changes an internal implementation, so it's only a patch release.

```suggestion
## 7.2.1 - TBD
```